### PR TITLE
Fix PR-answer sample

### PR DESCRIPTION
**Pull request by KaptenJansson**
**Description**
Update the track/stream handling + camelCase for localStream.

**Purpose**
Fix broken pr-answer sample page

### CODE REVIEW

**Comment by KaptenJansson** [comment]
Fixes https://bugs.chromium.org/p/chromium/issues/detail?id=1040913

**Review by MirkoBonadei — APPROVED** [review]
LGTM

File: src/content/peerconnection/pr-answer/js/main.js [hunk 1]
```diff
@@ -24,7 +24,9 @@ hangUpButton.disabled = true;
 
 let pc1 = null;
 let pc2 = null;
-let localstream;
+let localStream;
+const remoteStream = new MediaStream()
```
Comment (MirkoBonadei) [inline src/content/peerconnection/pr-answer/js/main.js]: Missing ;

**Review by MirkoBonadei — APPROVED** [review]
Still LGTM % comment.

File: src/content/peerconnection/pr-answer/js/main.js (hunk 1 above)
Comment (KaptenJansson) [reply src/content/peerconnection/pr-answer/js/main.js]: Fixed. Lets now see if travis is happy :)

**Pull request merged**